tools/checkpoint_conversion/convert_gpt_neox_checkpoints.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import os
+
+import requests
+import tensorflow as tf
+from transformers import AutoTokenizer
+from transformers import GPTNeoXModel
+
+from keras_nlp.models.gpt_neo_x.gpt_neo_x_backbone import GPTNeoXBackbone
+
+PRESET_NAME = "pythia-70m"
+PRESET = "EleutherAI/pythia-70m-deduped"
+EXTRACT_DIR = "./{}"
+
+
+extract_dir = EXTRACT_DIR.format(PRESET_NAME)
+if not os.path.exists(extract_dir):
+    os.makedirs(extract_dir)
+
+# Config.
+config_path = os.path.join(extract_dir, "config.json")
+response = requests.get(f"https://huggingface.co/{PRESET}/raw/main/config.json")
+# https://huggingface.co/EleutherAI/pythia-70m-deduped/raw/main/config.json
+
+open(config_path, "wb").write(response.content)
+
+# # Vocab.
```

Comment:
what's up with this commented out code? is pythia sentencepiece? delete or update